### ISSUE TITLE
Is it stable enough for production

### ISSUE THREAD

**Issue by ScreamZ** [vasern/vasern#52]
Hello, 

I will need to use whether Realm or Vasern for a production app.

Currently Realm isn't supporting Hermes engine, seems like Vasern do.

But I see it's pretty new, is is stable for production ?

Regards

**Comment by hieunc229** [comment]
Hi @ScreamZ, thanks for checking out Vasern. There are apps that use Vasern in production, for example ([Litely](https://play.google.com/store/apps/details?id=prj.adityasonel.litephone), internal apps for @kailashvele clients).

Vasern is definitely great for small apps, it's easy to get started. Though, the performance will decrease when dealing with the huge dataset (+100k records).

There is also a SQL database option for React Native that you might want to check out [WatermelonDB](https://github.com/Nozbe/WatermelonDB)

**Comment by ScreamZ** [comment]
Ohhh thanks @hieunc229 , thats look great.

Briefly, I'm making a Command App that will received Bluetooth data from an IOT device.

On connect to that IOT, the device will send me history of from measure that it took over time.

I started with async storage, but there is a limit of 6Mb on android by default, so I looked some sor LRU/FIFO things to handle cache.

I landed on realm, which looks great and performant and would allows me to run queries and handle saving for me, then I found Vasern.

If you're telling me it work, i'll probably try Vasern.
But that watermelon thing looks also so great… Damn too much choices
